### PR TITLE
Simplify LSP17ContractExtension Standard

### PR DESCRIPTION
**Pull request by YamenMerhi**
## What does this PR introduce ?

- Simplify the LSP17ContractExtension by not enforcing any behavior of storing selectors to addresses, events, talking about ERC725Y , etc ..
- Update different sections, Rational, Motivation, Abstract.
- Add the modification requirements in LSP0 and LSP9

### CODE REVIEW

File: LSPs/LSP-0-ERC725Account.md [hunk 1]
```diff
@@ -74,6 +74,20 @@ This function is part of the [LSP17] specification, with additional requirements
 
 - MUST be payable.
 - MUST emit a [`ValueReceived`] event if value was present.
```
Comment (CJ42) [inline LSPs/LSP-0-ERC725Account.md:76]:
```suggestion
- MUST emit a [`ValueReceived`] event if value was sent alongside some calldata.
```

File: LSPs/LSP-0-ERC725Account.md [hunk 2]
```diff
@@ -74,6 +74,20 @@ This function is part of the [LSP17] specification, with additional requirements
...
+- MUST return if the data sent to the contract is less than 4 bytes in length or if the first 4 bytes of the data are equal to 0.
```
Comment (CJ42) [inline LSPs/LSP-0-ERC725Account.md:77]:
```suggestion
- MUST return if the data sent to the contract is less than 4 bytes in length or if the first 4 bytes of the data are equal to `0x00000000`.
```

**Review by CJ42 — APPROVED** [review]
Amazing, that's very well explained. Great improvement! 💪

**Review by b00ste — APPROVED** [review]

**Review by skimaharvey — APPROVED** [review]
LGTM

**Review by frozeman — APPROVED** [review]

**Pull request merged**